src/rules/lowercase-name.ts

```diff
@@ -52,6 +53,32 @@ const testDescription = (argument: ArgumentLiteral): string | null => {
   return argument.quasis[0].value.raw;
 };
 
+const jestTopFunctioName = (
+  node: CallExpressionWithCorrectCalleeAndArguments,
+  counter: number,
+) => {
+  const description = testDescription(node.arguments[0]);
+  if (description === null) {
+    return null;
+  }
+
+  if (counter !== 1) {
+    return null;
+  }
+
+  const firstCharacter = description.charAt(0);
+
+  if (!firstCharacter) {
```

Comment:
Given that `charAt` is typed as returning a `string`, rather than `string | null`, I'd say it would be better to just check that `description` is truthy, rather than do this.

i.e change the earlier `description === null` condition to just `!description`, and remove this check.